core/point/point.go

```diff
@@ -21,3 +25,34 @@ func (p *Point) X() int64 { return p.x }
 
 // Y returns the y-value.
 func (p *Point) Y() int64 { return p.y }
+
+// Kilo Foxtrot addition below
+// Three point translation methods, one is an alternate
+
+// 1. '*Point' immutable int point, to SGF(String of two letters) Point
+func translateToSGFPt01(pt *Point) string {
+	sgfPtX := string(rune((pt.X()) + 97))
```

Comment:
instead of subtracting 97, how about storing as a const?

const aValue = int('a')

Then here you can just add pt.X() + aValue